airbyte-integrations/connectors/source-file/source_file/client.py

```diff
@@ -119,7 +119,9 @@ def _open(self, binary):
         elif storage in ("ssh://", "scp://", "sftp://"):
             user = self._provider["user"]
             host = self._provider["host"]
-            port = self._provider.get("port", 22)
+            # TODO: Remove int casting when https://github.com/airbytehq/airbyte/issues/4952 is addressed
+            # TODO: The "port" field in spec.json must also be changed
+            port = int(self._provider.get("port", 22))
```

Comment:
maybe in this part could be have a try/exception to return a meaningful error message in case people add a non-valid port value. If the issue you create is solve quickly probably is something not needed. WDYT?
```python
try:
   port = int(self._provider.get("port", 22))
except:
   raise ValueError("The value specify for the port is a not valid integer")
```

Reply:
I like it!

Reply:
```suggestion
            _port_value = self._provider.get("port", 22)
            try:
                port = int(_port_value)
            except ValueError as err:
                raise ValueError(f"{_port_value} is not a valid integer for the port") from err
```
@marcosmarxm this look okay?

Reply:
Yes!